csrankings-n.csv

```diff
@@ -356,6 +357,7 @@ Nicholas LaRacuente,Indiana University,https://luddy.indiana.edu/contact/profile
 Nicholas Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos,Dvp7fNoAAAAJ
 Nicholas Mattei,Tulane University,http://www2.tulane.edu/sse/cs/faculty/nicholas-mattei.cfm,bUAYDkIAAAAJ
 Nicholas Miles Gibbins,University of Southampton,https://www.ecs.soton.ac.uk/people/nmg,umB6pHIAAAAJ
+Nicholas Paul Sheppard,Western Sydney University,https://www.westernsydney.edu.au/staff_profiles/WSU/doctor_nicholas_sheppard,NOSCHOLARPAGE

```

Comment:
Partial appointment, professor of practice - not eligible for inclusion.

Reply:
I have removed that item as requested. Many thanks.